src/js/models/maps/AssetCategory.js

```diff
@@ -0,0 +1,127 @@
+"use strict";
+
+define([
+  "backbone",
+  "models/maps/Map",
+  "collections/maps/MapAssets",
+  "common/IconUtilities",
+], function (Backbone, MapModel, MapAssets, IconUtilities) {
+  /**
+   * @classdesc A AssetCategory Model contains metadata about the category, like a label and an icon.
+   * @classcategory Models/Maps
+   * @class AssetCategory
+   * @name AssetCategory
+   * @extends Backbone.Model
+   //  TODO: yvonneshi - update
+   //  * @since x.x.x
+   * @constructor
+   */
+  const AssetCategory = Backbone.Model.extend(
+    /** @lends AssetCategory.prototype */ {
+      /**
+       * The name of this type of model
+       * @type {string}
+       */
+      type: "AssetCategory",
+
+      /**
+       * Default attributes for AssetCategory models
+       * @name AssetCategory#defaults
+       * @type {Object}
+       * @property {string} label A user friendly name for this category, to be displayed
+       * in a map.
+       //  TODO: yvonneshi - Find default icon
+       * @property {string} icon
```

Comment:
@hngrtr Is there a default icon we should use for layers that don't have an icon? Or just leave it blank?

Same comment in Sketch here: https://www.sketch.com/s/eea6a4f3-5c59-40e6-bf7f-64edc511e4cb/p/ED8A5F3A-9D1C-4FD2-A60D-2077C041E444/canvas?annotation=b837a676-88df-4011-bff4-39f46a36737c

Reply:
Added default icon as discussed in https://www.sketch.com/s/0a63ad97-b63a-4a87-8ebf-e975a448754a/p/ED8A5F3A-9D1C-4FD2-A60D-2077C041E444/canvas?annotation=a267ca24-c25e-4a95-bf50-ecbbe63d5946